wfe/wfe.go

```diff
@@ -623,12 +624,16 @@ func (wfe *WebFrontEndImpl) UpdateAccount(
 		ID:  existingAcct.ID,
 	}
 
-	newAcct.Contact = updateAcctReq.Contact
-	// Verify that the contact information provided is supported & valid
-	prob = wfe.verifyContacts(newAcct.Account)
-	if prob != nil {
-		wfe.sendError(prob, response)
-		return
+	if updateAcctReq.Status == "deactivated" {
```

Comment:
I think it would be worth returning a `probs.Malformed` instance if the user POST's an update with `Status != "deactivated"` to be a little bit more strict here.

The way you have it implemented now will ignore an invalid status in an update which isn't unreasonable, but I'd like to shake out as many client errors in Pebble as possible. 

It looks like Boulder is generous and does similar: it only checks that the update status != the existing status: https://github.com/letsencrypt/boulder/blob/699c7e4c4448bf70cd2410504684cc60737cd504/wfe/wfe.go#L1138-L1153 which is fine because ultimately the SA ignores the update status and explicitly uses the deactivated status in `sa.DeactivateRegistration`: https://github.com/letsencrypt/boulder/blob/699c7e4c4448bf70cd2410504684cc60737cd504/sa/sa.go#L1357